Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM node:8.10-alpine AS static
+FROM node:11.7-alpine AS static
 WORKDIR /opt/scaife-viewer/src/
-COPY package.json package-lock.json .babelrc ./
-RUN npm install
+COPY package.json package-lock.json ./
+RUN npm ci
+COPY webpack.config.js .babelrc ./
 COPY ./static static
-RUN npm run build:prod
+ARG FORCE_SCRIPT_NAME
+RUN npm run build
```

Comment:
I'm a fan of running tests/linters in the multistage build.

If you add `RUN npm run lint` just above `npm run build` and add `"lint": "eslint ./static/src",` to the script section to *package.json*, then then build will break if the liner commands fails.